corehq/apps/users/models.py

```diff
@@ -1339,7 +1340,15 @@ def change_username(self, username):
         self.username = username
         self.save()
 
+    @classmethod
+    def save_docs(cls, docs, **kwargs):
+        utcnow = datetime.utcnow()
+        for doc in docs:
+            doc['last_modified'] = utcnow
+        super(CouchUser, cls).save_docs(docs, **kwargs)
+
```

Comment:
i think we should also make sure `bulk_save` gets updated too since i think it will still point to the old version: https://github.com/dimagi/couchdbkit/blob/jsonobject/couchdbkit/schema/base.py#L159